### ISSUE TITLE
Unhandled exception: Crash on Startup

### ISSUE THREAD

**Issue by TadhgEomonn** [DazedNConfused-/CDDA-Game-Launcher#49]
* Description: [Enter what you did and what happened]
* Version: 1.6.5
* OS: Windows-10-10.0.19043-SP0 (64-bit)
* Type: `<class 'IndexError'>`
* Value: list index out of range
* Traceback:
```
  File "C:\Users\Timothy\AppData\Local\Programs\CDDA Game Launcher\cddagl\ui\views\main.py", line 1379, in showEvent
    self.refresh_builds()
  File "C:\Users\Timothy\AppData\Local\Programs\CDDA Game Launcher\cddagl\ui\views\main.py", line 3235, in refresh_builds
    self.refresh_changelog()
  File "C:\Users\Timothy\AppData\Local\Programs\CDDA Game Launcher\cddagl\ui\views\main.py", line 3274, in refresh_changelog
    msg = body[3]

```

**Comment by Operative-Light** [comment]
Also getting this exact issue. Only started happening tonight

**Comment by SirClarke** [comment]
Yup same here sadly

**Comment by NeroPrime** [comment]
Same.

Description: Installed and used the launcher for the first time.
CDDA Game Launcher version: 1.6.5
OS: Windows-10-10.0.22621-SP0 (64-bit)
Type: <class 'IndexError'>
Value: list index out of range
Traceback:
File "D:\Games\CDDA Game Launcher\cddagl\ui\views\main.py", line 1379, in showEvent

self.refresh_builds()
File "D:\Games\CDDA Game Launcher\cddagl\ui\views\main.py", line 3235, in refresh_builds
self.refresh_changelog()
File "D:\Games\CDDA Game Launcher\cddagl\ui\views\main.py", line 3274, in refresh_changelog
msg = body[3]

**Comment by TravisLite** [comment]
I'm also getting this error.

**Comment by animation303** [comment]
same error, seems the cdda repository may have changed how the changelogs are stored or displayed

temporary fix:

```
open CDDA Game Launcher\cddagl\ui\views\main.py

go to line 3274

change msg = body[3] to msg = body[2]

save

open CDDA Game Launcher\cddagl\ui\views\__pycache__

delete all files

rerun launcher
```

the changelog section will no longer work, but you should be able to update again

**Comment by SirClarke** [comment]
Thanks, animation303 that worked for now

**Comment by TravisLite** [comment]
@DazedNConfused- Do you think you could take a look at this? Displaying the experimental changelog was one of the big draws of your update!

**Comment by DazedNConfused-** [comment]
Thanks for bringing this into my attention. Oddly enough, I didn't see any email notification until I was tagged just no (I should have a look into the Repository's notification settings as well...)

Will take a look at this. Thanks again!

**Comment by DazedNConfused-** [comment]
Should be fixed in v1.6.6

Please let me know if something else, related or not, comes up. Feel free to tag me for good measure, that way I also get an email.
